packages/core/src/render3/query.ts

```diff
@@ -33,16 +36,9 @@ export interface QueryPredicate<T> {
   type: Type<T>|null;
 
   /**
-   * If selector then contains the selector parts where:
-   * - even index:
```

Comment:
why delete all of this text?

Reply:
Because I've changed the proposed format for a selector. We can only query for names, we can't use CSS selectors in the current Angular. 

Supporting CSS selectors would require more complex structure (to support negation, commas etc.). I don't think we want to introduce this complexity now given that it is not present in the current Angular. And I would _love_ proper design doc if we want to add CSS selectors for queries.